modules/release-notes/pages/relnotes.adoc

```diff
@@ -22,14 +22,70 @@ See xref:install:install-platforms.adoc[Supported Platforms] for the complete li
 
 The notable fixes in this release are listed below.
 
+==== Analytics Service
+
+[#table_fixedissues_v601-analytics,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-32435[MB-32435^]
+| *Summary*: Fixed an intermittent rebalance failure observed when upgrading the Analytics node.
+|===
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v601-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-32299[MB-32299^]
+| *Summary*: XDCR failed to replicate files larger than the network usage limit. This has been fixed so the bandwidth throttler can handle large documents with sizes greater than the usage limit.
+
+| https://issues.couchbase.com/browse/MB-32221[MB-32221^]
+| *Summary*: Uncompressed replication from 6.0 nodes to 5.0 nodes failed due to partial information being returned from the parts. This has been fixed to enable accurate detection of compression error code.
+
+| https://issues.couchbase.com/browse/MB-32220[MB-32220^]
+| *Summary*: Fixed an issue where replication stops and throws an exception due to a nil error.
+
+| https://issues.couchbase.com/browse/MB-32219[MB-32219^]
+| *Summary*: Fixed an issue where the XDCR log level setting was not retained when upgrading Couchbase Server from version 4.x to newer version.
+
+| https://issues.couchbase.com/browse/MB-32042[MB-32042^]
+| *Summary*: Fixed an issue where encrypted XDCR was incorrectly using port 8091.
+
+| https://issues.couchbase.com/browse/MB-31857[MB-31857^]
+| *Summary*: An error was thrown when DNS alternate addresses were used. This has been fixed so utilities return the hostname even if they can't find the port numbers.
+
+| https://issues.couchbase.com/browse/MB-31764[MB-31764^]
+| *Summary*: The unit of interval for XmemSelfMonitorInterval was incorrectly set to millisecond instead of second, causing an increase in the frequency of Xmem self monitor and the spurious Xmem is stuck error. This has been fixed.
+|===
+
 ==== Data Service
```

Comment:
A number of MBs are missing from the list for the data service - see https://issues.couchbase.com/issues/?jql=project%20%3D%2010010%20AND%20fixVersion%20%3D%2015522%20AND%20component%20in%20(memcached%2C%20couchbase-bucket)%20ORDER%20BY%20key%20ASC%2C%20priority%20DESC

For the ones also fixed in 5.5.x you can just copy the summary from 5.5.3. 

The other ones which are not already in 5.5.3 are as follows, along with a proposed summary:

* https://issues.couchbase.com/browse/MB-32078 *Summary*: If an Append/Prepend request contends with another request at the server, it may not be automatically retried on the server (as expected) and instead will return EEXISTs back to the application.

* https://issues.couchbase.com/browse/MB-32364 *Summary*: Sub-Document API single-path mutation ignores expiry if docflags included in extras.

* https://issues.couchbase.com/browse/MB-32589 *Summary*: KV-Engine diagnostic stat groups incorrectly increase bucket mem_used stat.

Reply:
Thanks @daverigby. MB-32589 is a private MB and hence won't be included in the release notes. Please let me know if there is a public ticket that we can use instead.